app/src/main/java/com/movies/presentation/diff_util/DiffUtil.kt

```diff
@@ -1,4 +1,4 @@
-package com.movies.presentation.base.diff_util
+package com.movies.presentation.diff_util
```

Comment:
you have view package in the presentation you can move diffutil into the view or you can have diffutil inside the common

Reply:
moved it in the utils package